### PR TITLE
Updating docs with v0.10 api

### PR DESCRIPTION
**Pull request by anpaz-msft**

### CODE REVIEW

File: api/qsharp/microsoft.quantum.primitive.multix.yml [hunk 1]
```diff
@@ -3,11 +3,7 @@ uid: microsoft.quantum.primitive.multix
 name: MultiX
 type: operation
 namespace: Microsoft.Quantum.Primitive
-summary: |-
-  > [!WARNING]
-  > Deprecated
-
-  Please use @"microsoft.quantum.canon.applytoeachca".
+summary: MultiX has been deprecated. Please use @"ApplyToEachCA(X, _)" instead.
```
Comment (cgranade) [inline api/qsharp/microsoft.quantum.primitive.multix.yml]: @bettinaheim, can you double-check this? Looks like it's not quite the right output.

File: api/qsharp/microsoft.quantum.canon.applyquantumfouriertransformbe.yml [hunk 2]
```diff
@@ -3,11 +3,7 @@ uid: microsoft.quantum.canon.applyquantumfouriertransformbe
 name: ApplyQuantumFourierTransformBE
 type: operation
 namespace: Microsoft.Quantum.Canon
-summary: |-
-  > [!WARNING]
-  > Deprecated
-
-  This operation has been removed.
+summary: ApplyQuantumFourierTransformBE has been deprecated. Please use @"ApplyQuantumFourierTransform(BigEndianAsLittleEndian(qs))" instead.
```
Comment (cgranade) [inline api/qsharp/microsoft.quantum.canon.applyquantumfouriertransformbe.yml]: Similarly here, @bettinaheim, I would appreciate a quick double-check.

File: api/qsharp/microsoft.quantum.canon.applyripplecarrycomparatorbe.yml [hunk 3]
```diff
@@ -3,11 +3,7 @@ uid: microsoft.quantum.canon.applyripplecarrycomparatorbe
 name: ApplyRippleCarryComparatorBE
 type: operation
 namespace: Microsoft.Quantum.Canon
-summary: |-
-  > [!WARNING]
-  > Deprecated
-
-  This operation has been removed.
+summary: ApplyRippleCarryComparatorBE has been deprecated. Please use @"CompareUsingRippleCarry(BigEndianAsLittleEndian(x), BigEndianAsLittleEndian(y), output)" instead.
```
Comment (cgranade) [inline api/qsharp/microsoft.quantum.canon.applyripplecarrycomparatorbe.yml]: This also looks like it's not actually a valid link?

File: api/qsharp/microsoft.quantum.canon.measureintegerbe.yml [hunk 4]
```diff
@@ -3,11 +3,7 @@ uid: microsoft.quantum.canon.measureintegerbe
 name: MeasureIntegerBE
 type: operation
 namespace: Microsoft.Quantum.Canon
-summary: |-
-  > [!WARNING]
-  > Deprecated
-
-  This operation has been removed.
+summary: MeasureIntegerBE has been deprecated. Please use @"MeasureInteger(BigEndianAsLittleEndian(target))" instead.
```
Comment (cgranade) [inline api/qsharp/microsoft.quantum.canon.measureintegerbe.yml]: One more non-link, it looks like.

**Comment by KittyYeungQ** [comment]
@cgranade Is this one outdated and should be closed?

**Comment by cgranade** [comment]
> @cgranade Is this one outdated and should be closed?

It does appear to be, yes.

**Pull request closed without merging**